README.md

```diff
@@ -43,40 +43,110 @@ All these files generated are stored in inst/test files in your test package.
 
 ```R
-RcppDeepState::deepstate_harness_analyze_one("~/R/RcppDeepState/inst/testpkgs/testSAN")
+result = RcppDeepState::deepstate_harness_analyze_pkg("~/R/RcppDeepState/inst/testpkgs/testSAN")
+result
 ```
 
-Example Output for function src/read_out_of_bound.cpp:
+The result contains a data table with three colums: binary.file,inputs,logtable
 
 ```R
-Input parameter - rbound
-Read 1 item
-[1] 318916283
+> head(rd,2)
+                                                                                                                                                                                         binaryfile
+1: /home/akolla/R/x86_64-pc-linux-gnu-library/3.6/RcppDeepState/testpkgs/testSAN/inst/testfiles/rcpp_read_out_of_bound/rcpp_read_out_of_bound_output/0001957a365ef90344a992e32cc2d49d4aedf572.crash
+2: /home/akolla/R/x86_64-pc-linux-gnu-library/3.6/RcppDeepState/testpkgs/testSAN/inst/testfiles/rcpp_read_out_of_bound/rcpp_read_out_of_bound_output/0001b796162c8cd4b00f4b7ccf165b55b566cfce.crash
+      inputs          logtable
+1: <list[1]> <data.table[2x5]>
+2: <list[1]> <data.table[1x5]>
+> 
+
+
+```
+The inputs column contains all the inputs that are passed: 
+
+```R
+> head(rd$inputs,2)
 [[1]]
-          kind                    msg                    errortrace
-1: InvalidRead Invalid read of size 4 src/read_out_of_bound.cpp : 7
-                  address trace
-1: No address trace found    NA
+[[1]]$rbound
+[1] 437585945
+
 
 [[2]]
-                kind
-1: Leak_PossiblyLost
-                                                                               msg
-1: 1,275,665,132 bytes in 1 blocks are possibly lost in loss record 1,279 of 1,279
-                      errortrace                address trace
-1: src/read_out_of_bound.cpp : 6 No address trace found    NA
+[[2]]$rbound
+[1] -519122509
 
 ```
+The logtable has the data table with list of errors:
+
+```R
+> head(rd$logtable,2)
+[[1]]
+             problem
+1:       InvalidRead
+2: Leak_PossiblyLost
+                                                                           message
+1:                                                          Invalid read of size 4
+2: 1,750,343,780 bytes in 1 blocks are possibly lost in loss record 1,279 of 1,279
+                       file.line
+1: src/read_out_of_bound.cpp : 7
+2: src/read_out_of_bound.cpp : 6
+                                                                                        address.msg
+1: Address 0xc21df234 is 1,750,344,180 bytes inside a block of size 1,750,347,680 in arena "client"
+2:                                                                                 No Address found
+            address.trace
+1: No Address Trace found
+2: No Address Trace found
+
+[[2]]
```

Comment:
instead of a list, can you please return a data table? (more user friendly)

Reply:
I am returning a data table(columns: binary file, inputs, logtable) from the analyze_pkg function but the logtable(column of the result) is a list of data tables.
should I return a data table of data tables instead?

```
> head(rd,3)
                                                                                                                                                                                         binaryfile
1: /home/akhila/R/x86_64-pc-linux-gnu-library/3.6/RcppDeepState/testpkgs/testSAN/inst/testfiles/rcpp_read_out_of_bound/rcpp_read_out_of_bound_output/0000315982c3495464a54940ff9b1b02b02f1789.crash
2: /home/akhila/R/x86_64-pc-linux-gnu-library/3.6/RcppDeepState/testpkgs/testSAN/inst/testfiles/rcpp_read_out_of_bound/rcpp_read_out_of_bound_output/0000f834b0318349df532962a673f8708707f4ba.crash
3: /home/akhila/R/x86_64-pc-linux-gnu-library/3.6/RcppDeepState/testpkgs/testSAN/inst/testfiles/rcpp_read_out_of_bound/rcpp_read_out_of_bound_output/0001ebbc49f8e55072a0517fd4443d13d14bcf70.crash
      inputs          logtable
1: <list[2]> <data.table[1x5]>
2: <list[2]> <data.table[1x5]>
3: <list[2]> <data.table[1x5]>


```

Reply:
in the example maybe just show one of the logtables, or all of them, 
```r
rd$logtable[[1]]
do.call(rbind, rd$logtable)
```

Reply:
using rd$logtable[[1]] and returning the data table.